packages/nouns-webapp/src/components/NounImageVoteTable/index.tsx

```diff
@@ -0,0 +1,42 @@
+import { StandaloneNounCircular } from '../../components/StandaloneNoun';
+import { BigNumber as EthersBN } from 'ethers';
+import classes from './NounImageVoteTable.module.css';
+import { GrayCircle } from '../GrayCircle';
+
+interface NounImageVoteTableProps {
+  nounIds: string[];
+}
+const NOUNS_PER_VOTE_CARD = 15;
```

Comment:
nit: let's add `DESKTOP` to var name so we know that it's meant to be 15 on desktop only